src/components/Web3Status/index.tsx

```diff
@@ -197,6 +184,8 @@ function Web3StatusInner() {
   } = useDerivedSwapInfo()
   const validSwapQuote = getIsValidSwapQuote(trade, tradeState, swapInputError)
   const navbarFlag = useNavBarFlag()
```

Comment:
This can be done in one line

    const navbarFlagEnabled = useNavBarFlag() === NavBarVariant.Enabled

Reply:
thanks sg, done